content/articles/pathlib-vs-os-modules-for-file-and-directory-system-operation/index.md

```diff
@@ -0,0 +1,910 @@
+
+- Utilizing pathlib and os modules to manipulating a file and a directory path.
+- Implementing object-oriented programming and functional programming in file system operation.
+- Converting a file's metadata to basic readable information.
+- Manipulating relative path and absolute path.
+- Understanding the scenario where os and pathlib modules can be independently fit in.
+
+### Contents
+- [Introduction](#introduction)
+- [Os module](#os-module)
+- [Pathlib module](#pathlib-module)
+- [Highlighting the practical difference between os and pathlib module](#highlighting-the-practical-difference-between-os-and-pathlib-module)
+- [Querying current path using os and pathlib](#querying-current-path-using-os-and-pathlib)
+- [Navigating to home directory](#navigating-to-home-directory)
+- [Listing a directory path content](#listing-a-directory-path-content)
+- [Creating a new directory and file](#creating-a-new-directory-and-files)
+- [Checking for an existing file or directory](#checking-for-an-existing-file-or-directory)
+- [Renaming a directory and file](#renaming-a-directory-or-file)
+- [Joining path components together](#joining-path-component-together)
+- [Querying path to retrieve a final file or directory name](#querying-path-to-retrieve-a-final-file-or-directory-name)
+- [Removing a directory](#removing-a-directory)
+- [Removing a file](#removing-a-file)
+- [Retrieving and converting a file information](#retrieving-and-converting-a-file-information)
+- [Querying a file extension and name](#querying-a-file-extension-and-name)
+- [Theoretical difference between pathlib and Os](#theoretical-difference-between-pathlib-and-os)
+- [Wrapping Up](#wrapping-up)
+
+### Introduction
+File System Operation can be characterized as the example or technique that can be used to control or construct files and directories and there are two different ways of working with file system operation, which are functional dependency and object-oriented concept. This tutorial practically highlights the different scenarios to differentiate between Pathlib and Os.
+
+#### Os module
+The OS (operating system) module is a different operating system interface that functionally renders a convenient way of working with files and directories with a simple function call, additionally, OS empowers us to deal with I/O tasks straightforwardly.
+
+#### Pathlib module
+The Pathlib module is an Object-Oriented file system activity that offers classes representing filesystem paths with semantics fitting for various operating systems.
+
+### Highlighting the practical difference between os and pathlib modules
+In this section we will practically illustrate the semantic differences between pathlib and os modules using comparable code snippets to help you determine where every module fit in. Additionally, note that every demonstration requires the importation of the two modules.
+
+### Querying current path using os and pathlib
```

Comment:
`###` -> `####`

Do this change for the below sub-topics

Reply:
`OS and Pathlib`

Reply:
Okay i will do so.